src/views/standalone/network/InterfacesAndDevicesView.vue

```diff
@@ -775,14 +783,17 @@ function isDeviceConfigurable(deviceOrIface: DeviceOrIface) {
                           <span class="font-medium">RX: </span>
                           <span>{{ getRxBytes(device) || '-' }}</span>
                           <span v-if="device.stats?.rx_packets">
-                            ({{ device.stats.rx_packets }} pkts)</span
+                            ({{ formatPackets(device.stats.rx_packets) }} pkts)</span
```

Comment:
Is a null check on `device.stats.rx_packets` needed? This check is present for `tx_packets`

Reply:
That's a good question, initially I simply considered the original state of the view (which used the bool check only for `tx_packets`), but looking at the payload i'm not sure it's even needed, since I'm seeing it's always a number... I think we need to decide whether we simply want to show "0 pkts" for both fields or "- pkts" if no packets are sent/received. I'd personally use the first option

Reply:
Looking more carefully, the null check inside `{{ ... }}` is useless: the two `span`s won't appear because they contain a null check themselves: `<span v-if="device.stats?.rx_packets">` and `<span v-if="device.stats?.tx_packets">`. We can safely remove the null check on `tx_packets` at line 794